src/features/in-app-messenger/contextual/templates/PreviewTitleBodyTwoButtonsPopoutTemplate.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+/**
+ * This component is intended to be passed to TemplateContainer as templateComponent prop, like this:
+ *   <TemplateContainer templateID='my-template' templateComponent=MyTemplate/>
+ */
+
+import * as React from 'react';
+
+import Button from '../../../../components/button/Button';
+import Overlay from '../../../../components/flyout/Overlay';
+import MessagePreviewContent from '../../../message-preview-content/MessagePreviewContent';
+import type { Token } from '../../../../common/types/core';
+
+import { type PreviewTitleBodyTwoButtonsPopoutParams } from '../../types';
+
+import './styles/PreviewTitleBodyTwoButtonsPopoutTemplate.scss';
+
+type Props = {
+    apiHost: string,
+    contentPreviewProps?: ContentPreviewProps,
+    getToken: (folderID: string | number) => Promise<Token>,
+    onAction: Function,
```

Comment:
can you type this better?

Reply:
This require to add one more file to define MessageAction types